source/JsMaterialX/JsMaterialXGenShader/JsShaderNode.cpp

```diff
@@ -19,5 +19,9 @@ EMSCRIPTEN_BINDINGS(ShaderPort)
         .function("getVariable", &mx::ShaderPort::getVariable)
         .function("getType", &mx::ShaderPort::getType, ems::allow_raw_pointers())
         .function("getValue", &mx::ShaderPort::getValue)
+        .function("getPath", &mx::ShaderPort::getPath)
```

Comment:
Reflection API is still incomplete but this helps with some basics including getting a path back to get things like UI attributes etc.